### ISSUE TITLE
Loss Function

### ISSUE THREAD

**Issue by sdlpkxd** [hengchuan/RDN-TensorFlow#12]
Hi, thanks for your tensorflow implement .
I find that you use L2 loss in the model.py, it's different from the paper. Why we not use the L1 loss, can L2 loss bring some benefit for training ?

**Comment by hengchuan** [comment]
@sdlpkxd Hi, thanks for your feedback!
It was a mistake. Experiments shows that L1 loss leads to better performance as the paper described. I have correct the implement. Sorry for my carelessness .
